tests/suites/test_suite_psa_crypto_pake.function

```diff
@@ -537,7 +544,7 @@ exit:
 
 /* BEGIN_CASE depends_on:PSA_WANT_ALG_JPAKE */
 void ecjpake_setup(int alg_arg, int key_type_pw_arg, int key_usage_pw_arg,
-                   int primitive_arg, int hash_arg, int role_arg,
+                   int primitive_arg, int hash_arg, char *user_arg, char *peer_arg,
```

Comment:
```suggestion
                   int primitive_arg, int hash_arg, char *user_arg, char *peer_arg,
```
use rather data_t.

Reply:
With `data_t` I would have to provide bytes in hex as input. Instead `"client"` it would be `"636C69656E74"`. I think null-terminated string is better for this case.